lib/pure/os.nim

```diff
@@ -1647,8 +1647,11 @@ proc setFilePermissions*(filename: string, permissions: set[FilePermission]) {.
       var res2 = setFileAttributesA(filename, res)
     if res2 == - 1'i32: raiseOSError(osLastError(), $(filename, permissions))
 
-proc copyFile*(source, dest: string) {.rtl, extern: "nos$1",
-  tags: [ReadIOEffect, WriteIOEffect], noWeirdTarget.} =
+proc createSymlink*(src, dest: string) {.tags: [ReadDirEffect, WriteIOEffect], noWeirdTarget.}
+proc expandSymlink*(symlinkPath: string): string {.tags: [ReadIOEffect], noWeirdTarget.}
+
+proc copyFile*(source, dest: string, followSymlinks: bool = true) {.rtl, extern: "nos$1",
```

Comment:
```suggestion
proc copyFile*(source, dest: string, followSymlinks = true) {.rtl, extern: "nos$1",
```

Reply:
please always click 'resolve conversaion' when something was resolved (unless it's a pushback)